src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Assemblies/Ecma/EcmaAssembly.cs

```diff
@@ -55,7 +55,7 @@ protected sealed override AssemblyNameData[] ComputeAssemblyReferences()
                 AssemblyNameFlags flags = ar.Flags.ToAssemblyNameFlags();
                 data.Flags = flags;
                 data.Name = ar.Name.GetString(reader);
-                data.Version = ar.Version.AdjustForUnspecifiedVersionComponents();
+                data.Version = ar.Version.AdjustForUnspecifiedVersionComponents()!;
```

Comment:
Is this bang needed?

Reply:
Yes, the called method can return null values.